manifests/repo.pp

```diff
@@ -143,11 +143,11 @@
           require => Exec["import_gpg_${normalized_name}"],
         }
 
-        exec { "yum_make_cache_${repo_name}":
-          command => "yum -q makecache -y --disablerepo='*' --enablerepo='${normalized_name}'",
-          path    => '/usr/bin',
-          require => File[$normalized_name],
```

Comment:
Isn't the idea to ensure the latest cache has been build against the remote yumrepo?